.github/workflows/build-workflow.yml

```diff
@@ -61,28 +73,43 @@ jobs:
       - name: Enable toolchain via github action
         uses: dtolnay/rust-toolchain@1.63.0
         with:
-          targets: ${{ matrix.target }}
+          targets: ${{ matrix.job.target }}
 
-      - name: Build for ${{ matrix.target }}
-        run: bash -x ./ci/build_scripts/build.sh ${{ matrix.target }}
+      - name: Build for ${{ matrix.job.target }}
+        run: bash -x ./ci/build_scripts/build.sh ${{ matrix.job.target }}
 
       - name: Upload debian packages as zip
         # https://github.com/marketplace/actions/upload-a-build-artifact
         uses: actions/upload-artifact@v3
         with:
-          name: debian-packages-${{ matrix.target }}
-          path: target/${{ matrix.target }}/debian/*.deb
+          name: debian-packages-${{ matrix.job.target }}
+          path: target/${{ matrix.job.target }}/debian/*.deb
 
       - name: Upload sawtooth publisher as zip
         # https://github.com/marketplace/actions/upload-a-build-artifact
         uses: actions/upload-artifact@v3
         with:
-          name: sawtooth-publisher_${{ matrix.target }}
-          path: target/${{ matrix.target }}/release/sawtooth-publisher
+          name: sawtooth-publisher_${{ matrix.job.target }}
+          path: target/${{ matrix.job.target }}/release/sawtooth-publisher
```

Comment:
We will need to check if this test tool is still used. I think it's useless to maintain it.

Reply:
Yes it is currently used in the `ci_pipeline` workflow. But I have a roadmap item which will look at consolodating the workflows and removing unncessary artifacts.